### ISSUE TITLE
Django and react login with google

### ISSUE THREAD

**Issue by AREEG94FAHAD** [Tivix/django-rest-auth#645]
I am trying to use django rest auth with [react login with google](https://www.npmjs.com/package/react-google-login).

backend
In  my views.py file 
`class GoogleLogin(SocialLoginView):
    adapter_class = GoogleOAuth2Adapter `

In urls.py
`path('rest-auth/google/', GoogleLogin.as_view(), name='google_login') `

I also configured the database with google credentials 

Front part 

In authapt.js
`const googleLogin = async (accesstoken,code) => {
    const res = await axios.post(
      "http://localhost:8000/rest-auth/google/",
      {
        access_token: accesstoken,
        code
        
      }
    );
    console.log(res);
    return  res.status;
  }; `

In Logincomponent.jsx
`<GoogleLogin
          clientId="clientId"
          buttonText="LOGIN WITH GOOGLE"
          onSuccess={googleLogin  }
          onFailure={googleLogin}
        /> `

but when I try to login with my google account I got `xhr.js:177 POST http://localhost:8000/rest-auth/google/ 400 (Bad Request) ` i want to save the user in database any idea

**Comment by mateokurti** [comment]
I don't think this repo is taking any update. I suggest you to take a look at https://github.com/iMerica/dj-rest-auth
